### ISSUE TITLE
Address -Wsometimes-uninitilized warnings

### ISSUE THREAD

**Issue by jamienoss** [spacetelescope/hstcal#36]

**Comment by jamienoss** [comment]
The above PR (#37) outlines the offending cases where variables are sometimes used uninitialized.

**Comment by pllim** [comment]
You closed that PR. Do you still intend to get it merged?

**Comment by jamienoss** [comment]
@pllim No, at this point it can simply serve as an indication of the particular lines of code under question (I will add some inline comments to aid).

**Comment by pllim** [comment]
Duplicate of #35
